src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -51,7 +61,8 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Map<String, Long> getCountByRoadSurfaceCondition7(){
-        return null;
+        return roadAccidentList.stream()
+                .collect(Collectors.groupingBy(RoadAccident::getRoadSurfaceConditions, Collectors.counting()));
```

Comment:
so you are familiar with java 8? :smile:  
I think I did not cover static methods invocations ...